### PR TITLE
Add doc about integration with Azure DevOps

### PR DESCRIPTION
**Pull request by jgsogo**
Content from https://jfrog.com/blog/accelerate-azure-devops-or-tfs-with-jfrog-artifactory-and-conan/

Shall we link back to it?

closes https://github.com/conan-io/docs/issues/994

image

### CODE REVIEW

File: integrations/azure_devops.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ * Run Conan commands
+ * Resolve Conan dependencies from remote Artifactory servers
+ * Push Conan packages to Artifactory
+ * Publish BuildInfo metadata
+ * Import a Conan configuration
+
+In this section we will show you how to add Conan tasks to your pipelines using the Artifactory
+Extension and push the generated buildinfo metadata to Artifactory where it can be used to track
+and automate your builds.
+
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
```

File: integrations/azure_devops.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Configuring DevOps Azure to use Artifactory with Conan
+------------------------------------------------------
+
+To use the Conan support provided by the JFrog Artifactory Extension you must
+`configure a self-hosted agent`_
+that will enable Conan builds for your Azure Pipelines environment. Afterward you can install
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
that will enable Conan builds for your Azure Pipelines environment. Afterwards you can install
```

File: integrations/azure_devops.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+the JFrog Artifactory Extension from the Visual Studio Marketplace and follow the installation
+instructions in the Overview.
+
+.. image:: ../images/azure_devops/azure_devops_1.png
+   :width: 800 px
+   :alt: Azure DevOps
+
+When completed, proceed to create builds and access buildinfo from within Azure DevOps or TFS.
+
+
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
```

File: integrations/azure_devops.rst [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      :alt: Azure DevOps
+
+ 3. In the resulting Update Authentication for Artifactory dialog, enter the required server and
+    credential information, and click OK.
+
+   .. image:: ../images/azure_devops/azure_devops_3.png
+      :width: 800 px
+      :alt: Azure DevOps
+
+
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
```

File: integrations/azure_devops.rst [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+   .. image:: ../images/azure_devops/azure_devops_5.png
+      :width: 800 px
+      :alt: Azure DevOps
+
+ 5. Configure the Conan command for the task.
+
+   .. image:: ../images/azure_devops/azure_devops_6.png
+      :width: 800 px
+      :alt: Azure DevOps
+
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
```

File: integrations/azure_devops.rst [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+Continue to add Conan tasks as you need for each pipeline.
+
+
```
Comment (danimtb) [inline integrations/azure_devops.rst]:
```suggestion
```

**Comment by uilianries** [comment]
BTW should we add a section for Azure DevOps like we did for Gitlab, Travis?

**Comment by danimtb** [comment]
@uilianries what do you mean? This PR creates a section in the Integrations one. Is there any other place we should link this info to?

**Comment by uilianries** [comment]
I read about Artifactory + Azure DevOps, but not how to write [azure-pipelines.yml](https://github.com/bincrafters/conan-templates/blob/e332392afb434f28bac8b668dd160a9b4598d0cb/azure-pipelines.yml) recipe.

**Comment by danimtb** [comment]
@jgsogo seems you started this PR from develop and it is not something associated to the next release, so it should target master branch.

**Comment by uilianries** [comment]
> Do we need that azure-pipelines.yml file to use Azure Devops?

Yes, it works like Travis, Gitlab or any other. I could create a section explaining about or just append on this PR after to be merged.

**Comment by jgsogo** [comment]
Ok, @danimtb, now the PR is from `master` to `master`

**Comment by uilianries** [comment]
I didn't try console commands yet, but it should be straightforward as you said. That file was created reading Azure documentation.

File: integrations/azure_devops.rst [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+Extension and push the generated buildinfo metadata to Artifactory where it can be used to track
+and automate your builds.
+
+Configuring DevOps Azure to use Artifactory with Conan
+------------------------------------------------------
+
+To use the Conan support provided by the JFrog Artifactory Extension you must
+`configure a self-hosted agent`_
+that will enable Conan builds for your Azure Pipelines environment. Afterwards you can install
+the JFrog Artifactory Extension from the Visual Studio Marketplace and follow the installation
```
Comment (SSE4) [inline integrations/azure_devops.rst]: link to the Extension would be helpful
are you talking about this one? https://marketplace.visualstudio.com/items?itemName=JFrog.JFrog
online search doesn't show others by keywords `JFrog` or `Artifactory`, but description talks about `XRay`
in Visual Studio itself, I get `No items found`.

Comment (jgsogo) [reply integrations/azure_devops.rst]: The link is in the first paragraph, first line, of the page, under the _"JFrog Artifactory Extension for Azure DevOps and TFS"_

File: integrations/azure_devops.rst [hunk 8]
```diff
@@ -0,0 +1,142 @@
+.. _azure_devops:
+
+
+|azure_logo| Azure DevOps
```
Comment (memsharded) [inline integrations/azure_devops.rst]: Please add "Microsoft", "TFS" and "VSTS" to the title.

File: integrations/azure_devops.rst [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+The support for Conan now in the JFrog Artifactory Extension helps you perform the following
+tasks in Azure DevOps or TFS:
+
+ * Run Conan commands
+ * Resolve Conan dependencies from remote Artifactory servers
+ * Push Conan packages to Artifactory
+ * Publish BuildInfo metadata
+ * Import a Conan configuration
+
+In this section we will show you how to add Conan tasks to your pipelines using the Artifactory
```
Comment (memsharded) [inline integrations/azure_devops.rst]: using the "Artifactory/Conan TFS Extension" (not an extension of Artifactory, but an extension to TFS)

File: integrations/azure_devops.rst (hunk 8 above)
Comment (jgsogo) [reply integrations/azure_devops.rst]: From [this link](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=vsts): _"Azure DevOps Services and Azure DevOps Server were formerly named Visual Studio Team Services (VSTS) and Visual Studio Team Foundation Server (TFS)"_, the title could be (not fair enough):

"Microsoft's Azure DevOps (formerly known as RFS or VSTS)"

**Pull request merged**